netket/optimizer/qgt/default.py

```diff
@@ -42,6 +42,12 @@ def _is_dense_solver(solver: Any) -> bool:
 
 
 def default_qgt_matrix(variational_state, solver=False, **kwargs):
+    """
+    if variational_state is an ExactState, this performs the exact summation over the whole Hilbert space and returns exact metric tensor
+    """
+    from netket.vqs import ExactState
```

Comment:
Should be a top-level import.

Reply:
Hmm, no... I wouldn't risk it.
I think there's an import loop there if we do that.

Reply:
Ah, because of `.quantum_geometric_tensor` requiring `QGTAuto`? Fair enough.